ml/README.md

```diff
@@ -1,6 +1,17 @@
-# Interface for the Machine Learning Mlinference service, wasmcloud:mlinference
+[![crates.io](https://img.shields.io/crates/v/wasmcloud-interface-mlinference.svg)](https://crates.io/crates/wasmcloud-interface-mlinference)&nbsp;
+<!-- [![TinyGo Version](https://img.shields.io/github/go-mod/go-version/wasmcloud/interfaces?label=TinyGo&filename=mlinference%2Ftinygo%2Fgo.mod)](https://pkg.go.dev/github.com/wasmcloud/interfaces/mlinference/tinygo) -->
```

Comment:
same nits.. we should just delete the stuff that we're not  planning on rendering. I also don't know what the HTML comments will do on crates.io or docs.rs